SDWebImage/Core/SDDiskCache.m

```diff
@@ -285,6 +288,11 @@ - (void)moveCacheDirectoryFromPath:(nonnull NSString *)srcPath toPath:(nonnull N
         }
         // New directory does not exist, rename directory
         [self.fileManager moveItemAtPath:srcPath toPath:dstPath error:nil];
+        // disable iCloud backup
+        if (self.config.shouldDisableiCloud) {
```

Comment:
Does this needed ? Actually I don't know whether iCloud Backup remember the directory path, or the object (File Description or something) it pointed to

Reply:
Library/Caches isn't backed up by default, so it isn't needed. I kept it around in case people created disk cached in other directories.

Reply:
OK to merge in